packages/nimble-blazor/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -64,6 +64,12 @@ window.NimbleBlazor = {
         setData: async function (tableReference, data) {
             const dataObject = JSON.parse(data);
             tableReference.data = dataObject;
+        },
+        checkValidity: function (tableReference) {
+            return tableReference.checkValidity();
+        },
+        getValidity: function (tableReference) {
+            return tableReference.validity;
```

Comment:
Are we sure this is working? I would imagine this needs to be JSON.stringify before passing?

We need some playwright tests lol. Should we add something to the sample apps to run this? maybe like a button to press that runs it) as a manual test?

Reply:
I did see this working locally by putting some code in the sample app. I debated leaving that code in as part of this PR, but after some discussion with @atmgrifter00, we decided not to. But, we both completely agreed with the idea that we need playwright tests sooner rather than later.

I can go ahead and put my test code back into the sample app unless there are any objections.

Reply:
I don't feel strongly on way or another. If we do think we can start working on playwright tests and we capture this as a test that needs to be written that sounds fine. Long-term I don't think it's a particularly interesting in the example app, just (abusing?) it for a spot to put a manual test. If tested locally and y'all already agreed that we don't need it there sounds good to me 👍

Reply:
I'm planning to leave it as-is for now (nothing in the example to check validity), but I did manually double check that those functions are working now that I've made the rename from _rowId_ to _recordId_.